### ISSUE TITLE
getenv in Condor config is confusing in ,gangarc

### ISSUE THREAD

**Issue by drmarkwslater** [ganga-devs/ganga#494]
`Condor.getenv` is really a bool but is stored as a string (possibly for good reason as technically it is just mapped to a setting in the Condor submit file). This make things very awkward when setting it in the `.gangarc` as you would expect to just put:

```
[Condor]
getenv = True
```

But this fails because the config parser thinks this is a bool. I feel the best way around this is to change the schema to accept a bool as well but I need to make sure this won't break previous settings.

**Comment by twhyntie** [comment]
@drmarkwslater I guess being able to set it as a bool would be neater, but as an end user I don't particularly mind using either - as long as the docs reflect what's needed 😄

**Comment by drmarkwslater** [comment]
I've pushed an update to the docs to reflect this so the section at http://ganga.readthedocs.io/en/latest/UserGuide/UsingDifferentBackends.html#condor will reflect this issue.

**Comment by will-cern** [comment]
To be honest, I have no problem with this having to be a string in the config file, as long as the commented out example that is present in the auto-generated gangarc file you get reflects this. Up to now it said "#getenv = True" which leads you to believe it is a bool, not a string. If that is fixed, then I will be ok.
Interestingly though, even with the error I saw about the mismatch of types, the getenv property ended up being set to True in the jobs I submitted. So although it had a mismatch, it interpretted it correctly :-/
